pkg/cloud/awsprovider.go

```diff
@@ -1720,9 +1735,23 @@ func (aws *AWS) GetOrphanedResources() ([]OrphanedResource, error) {
 		if aws.isAddressOrphaned(address) {
 			cost := AWSHourlyPublicIPCost * timeutil.HoursPerMonth
 
+			desc := map[string]string{}
+			for _, tag := range address.Tags {
+				if tag.Key == nil {
+					continue
+				}
+				if tag.Value == nil {
+					desc[*tag.Key] = ""
+				} else {
+					desc[*tag.Key] = *tag.Value
+				}
```

Comment:
This will work, but I think it's also fine and simpler to just do:
```go
if tag.Key != nil && tag.Value != nil {
    desc[*tag.Key] = *tag.Value
}
```

I.e. no need to record an empty tag, right?

Reply:
Yeah, disregard. This will work just fine. I don't expect there to be many (any?) instances where we one is nil but the other isn't.